<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reset_Password</class>
 <widget class="QWidget" name="reset_Password">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*background-color: rgb(204, 204, 255);*/
/*background-color: rgb(100, 149, 237);*/
/*background-color: rgb(108, 117, 125);*/
/*background-color: rgb(218, 112, 214);*/
background-color: rgb(23, 130, 238);</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-3</y>
     <width>541</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Cooper Black</family>
          <pointsize>20</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 170, 127);</string>
        </property>
        <property name="text">
         <string>Reset Password</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="submitButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>280</y>
     <width>101</width>
     <height>47</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);
background-color: rgb(85, 85, 255);</string>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>431</width>
     <height>206</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="userIdlabel">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>User Id</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>118</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="newUserIdlineEdit">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>Please Enter New User Id</string>
        </property>
        <property name="placeholderText">
         <string>Enter User Id</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>58</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>   Security Question</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>48</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QComboBox" name="resetPasswordSecurityQuestioncomboBox">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <item>
           <property name="text">
            <string>------------Select Question------------</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>What was the name of your first pet ?</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>What is your mother's maiden name ?</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>What city were you born in ?</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>What is your favorite color ?</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>What is your childhood nickname ?</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>What was your favorite subject in school ?</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="resetPasswordSecurityQuestionlineEdit">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>Enter your answer here.....</string>
          </property>
          <property name="placeholderText">
           <string>Enter your answer here....</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="newPasswordLabel">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>New Password</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>88</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="newPasswordLineEdit">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>Please enter new password</string>
        </property>
        <property name="placeholderText">
         <string>Enter new password</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="confirmPasswordLabel">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Confirm Password</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="confirmPasswordLineEdit">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>Please enter confirm password</string>
        </property>
        <property name="placeholderText">
         <string>Enter confirm password</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
